SpatialGDK/Documentation/content/non-unreal-layers.md

```diff
@@ -129,48 +128,31 @@ Worker_RequestId SendSomeCommandRequest(Worker_EntityId TargetEntityId, Worker_C
 #### Receive data
 You could receive and deserialize a component update and command request in your Unreal project code in the following way:
 ```
-UCLASS()
-class SPATIALGDK_API SessionComponentCallbacks : UOpCallbackTemplate
+void ATPSGameMode::BeginPlay()
 {
-    GENERATED_BODY()
+    USpatialDispatcher* Dispatcher = Cast<USpatialNetDriver>(GetWorld()->GetNetDriver())->Dispatcher;
 
-public:
-    Worker_ComponentId GetComponentId()
-    {
-        return 1337;
-    }
+    Dispatcher->AddOpCallback(1337, [&](Worker_ComponentUpdateOp Op) {
```

Comment:
If we change the other signatures remember to change this one too.

Reply:
The worker SDK docs get away with '[&]' because it's in global scope. Here it's a bit iffy. Although I don't think this snippet actually needs to capture anything by ref. It looks like you might need to capture this, although it's probably better to get the world outside this scope.